nvme-print-json.c

```diff
@@ -4463,6 +4606,7 @@ static struct print_ops json_print_ops = {
 	.phy_rx_eom_log			= json_phy_rx_eom_log,
 	.ctrl_list			= json_nvme_list_ctrl,
 	.ctrl_registers			= json_ctrl_registers,
+	.ctrl_register			= json_ctrl_register,
```

Comment:
Please move the json bits into it's own patch. This just adds support for the `ctrl_register` callback.